packages/@lwc/style-compiler/src/__tests__/fixtures/z-index/expected.js

```diff
@@ -1,4 +1,4 @@
 function stylesheet(hostSelector, shadowSelector, nativeShadow) {
```

Comment:
Remove this fixture, meant to only run with `minify`. See in `packages/@lwc/style-compiler/src/__tests__/fixtures/z-index/config.json`.